setup.py

```diff
@@ -20,9 +20,8 @@
 ]
 
 testing_extras = [
-    'coverage',
-    'nose',
-    'nosexcover',
+    'pytest',
+    'pytest-cov',
```

Comment:
`pytest-cov` is not available e.g. in Debian stable (https://tracker.debian.org/pkg/python-pytest-cov) and hence causes addiitonal packaging problems on top of #139... Therefore, I would keep nosetest for now.